packages/devtools-ui-kit/src/module.ts

```diff
@@ -0,0 +1,46 @@
+import { fileURLToPath } from 'url'
+import { addComponentsDir, createResolver, defineNuxtModule, installModule } from '@nuxt/kit'
+import defu from 'defu'
+import { extendUnocssOptions } from './unocss'
+
+export { unocssPreset } from './unocss'
+
+const rPath = (p: string) => fileURLToPath(new URL(p, import.meta.url).toString())
+
+export interface ModuleOptions {
+  dev?: boolean
+  preset?: string
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: 'devtools-ui-kit',
+    configKey: 'devtoolsUIKit',
+  },
+  defaults: {
+    preset: rPath('./preset'),
+    dev: false,
+  },
+  async setup(options, nuxt) {
+    // Nuxt overrides
+    addComponentsDir({ path: rPath('./components/nuxt') })
```

Comment:
Shall we remove this to only keep components and overwrite the `NDarkToggle.vue` now we are focusing only for Nuxt?

Reply:
Yes since it support SSR, but so far for the devtools I think we can actually remove the color-mode and the component for Nuxt.

Once thing, could we give the Devtools color mode to the iframe somehow? 🤔

Reply:
It already inherits the color mode, if the iframe is in the same origin.

https://github.com/nuxt/devtools/blob/c5e9e174af368fe7ddf06f2fe47c6e162da0538b/packages/devtools/client/components/IframeView.vue#L60-L70